src/components/__tests__/axis.spec.js

```diff
@@ -0,0 +1,44 @@
+import axis from '../axis';
+
+describe('axis', () => {
```

Comment:
Add test for the case when `components` array do not exist on the layout

Reply:
Added